data/events/2024-los-angeles.yml

```
@@ -44,7 +44,9 @@ nav_elements: # List of pages you want to show up in the navigation of your page
   # - name: CFP
   #   url: https://www.socallinuxexpo.org/scale/21x/cfp
```

Up to you, but we recommend it for SEO etc.

```suggestion
  - name: CFP
    url: https://www.socallinuxexpo.org/scale/21x/cfp
```

---

We decided to have a page explaining how to submit a proposal before sending people to the SCALE website. Thank you for the recommendation, though. :)